docs/tools/cli.md

```diff
@@ -6,7 +6,20 @@ parts of the application.
 
 To help you create and delete new application modules, the starter kit comes with a custom Command Line Interface (CLI).
 
-You can view the help guide on using the CLI with by running:
+## Table of Contents
+
+* [First Use of the CLI](#first-use-of-the-cli)
+* [Selecting the Technology Stack with `choosestack`](#choosing-technology-stacks-with-choosestack)
+* [Deleting Features with `deletemodule`](#deleting-features-with-deletemodule)
```

Comment:
@AndSviat why we don't have a link for the 'addmodule' feature?

Reply:
@vsevolodbvd - I simply missed it. Fixed it with the latest commit.